lib/Default/AbstractSmrPlayer.class.php

```diff
@@ -2214,6 +2227,9 @@ public function killPlayer($sectorID) {
 		$this->setNewbieTurns(100);
 	}
 
+	/*
```

Comment:
Happy to add this unrelated comment, but might as well make it javadoc like the rest (double asterisk on the first line).

Reply:
unrelated in terms of the PR, correct?  ideally all the functions would have documentation?

Reply:
If the behavior of a method is fully and clearly described by the method name, then I don't think adding a doc is helpful (this especially applies to trivial getters/setters). I also think it's better to have no docs than docs that don't add any new information, e.g.
```php
/**
 * Get the start sector
 */
function getStartSector {
  ...
}
```
But yes, in general I am a proponent of thorough documentation. :)